packages/eds-tokens/figma/$metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "tokenSetOrder": [
+    "core/const",
+    "core/typography",
+    "core/sizing",
+    "core/spacing",
+    "core/color",
+    "readonly/compressed",
+    "readonly/spacing",
+    "readonly/tight",
+    "semantic/semantic",
+    "theme/default",
+    "theme/comfortable",
+    "theme/compact",
+    "theme/light",
+    "theme/dark",
+    "component/button",
+    "component/buttonTests/dimensions",
+    "component/buttonTests/figmaOptimised",
+    "component/buttonTests/beforeBorder",
+    "component/buttonTests/codeOptimised",
+    "semantic/shape",
+    "core/border"
+  ]
+}
```

Comment:
These files are generated by a plugin. So while I would normally add a newline, that’s just a temporary fix. The permanent fix would be to create a pull-request in the Token Studio repo, if we think it’s worth the effort.